tests/modules/nf-core/chewbbaca/createschema/main.nf

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env nextflow
+
```

Comment:
and remove pytests once nf-tests are integrated

Reply:
done with https://github.com/nf-core/modules/pull/4512/commits/09adb4179f3cca5d53e586c0e4bc57ae1c92c5c1